### DISCUSSION TITLE
posts/applying-giffgaff-sim

### DISCUSSION THREAD

**Discussion by giscus** [at-wr/blog#10] [Blog Posts Comments]
# posts/applying-giffgaff-sim

讲述了一些在中国申请 giffgaff 的历程以及一些使用其服务之须知内容...

https://wrye.dev/posts/applying-giffgaff-sim

**Comment by ChengRui24** [comment]
申请卡片的时候提示只能输入UK的postcode=。=

**Reply by at-wr** [reply]
可以多次尝试切换网络连接节点，直到出现 Country 的下拉框就可以填写中国地址了

**Comment by Oneideals** [comment]
我遇到了同样的问题

**Reply by at-wr** [reply]
试试切换节点呢，美国 IP 貌似是可以的

**Reply by Oneideals** [reply]
我全局美国，不太行，在尝试切换

**Comment by stan1233** [comment]
收件地址没有地方填写手机号，我把手机号写在地址的第二行，不知道能不能收到

**Reply by MisterHzX** [reply]
和我一样 我也是写到了地址的第二行  填了个  phone-xxxxxxxxxxx

**Comment by pluviophile8** [comment]
刚通过你的邀请链接申请了，如果我申请的这几张卡都激活的话，都是走你的AFF吗？

**Comment by Karos44** [comment]
请问下，我有办法查到单号吗？还是就干等？

**Comment by github-office** [comment]
收不到TG的验证码
